beta-src/src/state/game/game-api-slice.ts

```diff
@@ -342,6 +373,20 @@ const gameApiSlice = createSlice({
           highlightMapTerritoriesBasedOnStatuses(state);
         } else if (order.type === "move" && order.toTerritory !== null) {
           highlightMapTerritoriesBasedOnStatuses(state);
+        } else if (
+          order.method === "dblClick" &&
+          order.unitID !== clickData.payload.unitID
+        ) {
+          console.log("either a support move or a support hold");
```

Comment:
remove console.log

Reply:
This is still in progress, those are just placeholders to help. I will remove them in the next PR.